gcc_5-armv7hf/Dockerfile

```diff
@@ -13,7 +13,16 @@ ENV CC=arm-linux-gnueabihf-gcc-5 \
     LD=arm-linux-gnueabihf-ld \
     FC=arm-linux-gnueabihf-gfortran-5
 
-RUN sudo apt-get -qq update \
+RUN sudo dpkg --add-architecture armhf \
+    && sudo sed -i 's/deb\s/deb \[arch=amd64,i386\] /' /etc/apt/sources.list \
+    && PREFIX="deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports/" \
+    && SUFFIX="main restricted universe multiverse" \
+    && DEST="/etc/apt/sources.list.d/armhf.list" \
+    && echo "${PREFIX} xenial ${SUFFIX}" | sudo tee -a ${DEST} > /dev/null \
```

Comment:
I dont like to use pipe and echo on recipes, there is a recommendation about how to pipe on [Dockerfile Best Practices](https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/#using-pipes). However, I would recommend use only `printf` instead:

    printf "${PREFIX} xenial ${SUFFIX}" > ${DEST}

Reply:
The problem is that sudo does not apply to stdout redirections

Reply:
ops! I forgot about this! :man_facepalming:
Could you please update to use `printf` instead of `echo`?
Also, you need to add `set -o pipefail` as recommended by Docker community.

Or, you can provide the expected file in the folder and just copy when building the image. I think is more readable for humans.

Reply:
I made a new commit with the proposed changes